File: timetrials/src/main/java/fzero/timetrials/controller/RankingController.java
package fzero.timetrials.controller;

import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;

@Controller
@RequestMapping("rankings")
public class RankingController {
    @GetMapping("")
    public String getRankings() {
        return "ranking";
    }
}

File: FZero99-TT/src/main/kotlin/fZero/fZero99TT/controller/RankingController.kt
package fZero.fZero99TT.controller

import fZero.fZero99TT.applicationService.RankingApplicationService
import org.springframework.stereotype.Controller
import org.springframework.ui.Model
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping

@Controller
@RequestMapping("/rankings")
class RankingController(
    private val rankingApplicationService: RankingApplicationService
) {
    @GetMapping
    fun getRanking(model: Model): String {
        model.addAttribute("runsForRanking", rankingApplicationService.getAllTimes())
        return "ranking"
    }
}